### DISCUSSION TITLE
blog/do/2024-06-01-retrospect-pirogramming-19th/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#17] [Announcements]
# blog/do/2024-06-01-retrospect-pirogramming-19th/

moonshot

https://mingdodev.github.io/blog/do/2024-06-01-retrospect-pirogramming-19th/

**Comment by mikio999** [comment]
장고에 대해 언급해 주실 때 '노드나 스프링 공부를 할 때 대조군이 되어주는 느낌' 이라는 대목에서 인상이 깊었습니다! 19기의 꽃사슴 do님의 후기 잘 읽었습니다 ^^7

**Comment by hwanheejung** [comment]
기록 안해서 없어져버린 것들이 많다는 말이 너무 공감되네요..ㅠ 블로그 열시미 씁시당!
